packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -249,21 +216,27 @@ export class AtomicRefineModal implements InitializableComponent {
           )}
         </div>
         <slot name="facets"></slot>
-        <div
-          class="flex flex-col"
-          style={{
-            gap: 'var(--atomic-refine-modal-facet-margin, 20px)',
-            marginTop: isFacetElements
-              ? 'var(--atomic-refine-modal-facet-margin, 20px)'
-              : '',
-          }}
-        >
-          {automaticFacets}
-        </div>
+        <slot name="automatic-facets"></slot>
       </Fragment>
     );
   }
 
+  private getAutomaticFacetSlotContent() {
+    const isFacetElements =
+      this.bindings.store.getVisibleFacetElementsLength() > 0;
+    const amountToCollapseNext =
+      this.collapseFacetsAfter -
+      this.bindings.store.getVisibleFacetElementsLength();
+
+    const divSlot = document.createElement(
+      'atomic-automatic-facet-slot-content'
+    );
```

Comment:
Neat solution!

I wonder, would it be possible to use the normal automatic facet builder instead?